src/liquid/components/ld-typo/readme.md

```diff
@@ -595,6 +599,10 @@ Here are some examples on how you can apply different colors on headings:
 ### Graph
 ```mermaid
 graph TD;
+  docs-contributors --> ld-typo
```

Comment:
All the docs-related changes to these files should not be pushed. Before pushing, please run pnpm build, which *should* remove these changes.